src/main/webapp/js/datahandlers/scholarshiplistdatahandler.js

```diff
@@ -76,8 +76,9 @@ class ScholarshipListDataHandler {
   /**
    * @param {Object} item The schoarship object.
    * @returns {!Array<!Array<string>>} The formatted scholarship list item.
+   * @private
    */
-  formatListItem(item) {
+  formatListItem_(item) {
     const amount = (item[AMOUNT_PER_YEAR] || UNKNOWN);
```

Comment:
Are you shortening a number here with an ellipsis? I think it's best to display the whole number.